app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= form_with model: @task do |f| %>
+
+  <h2>Edit Task Form</h2>
+
+  <%= f.label :name %>
+  <%= f.text_field :name %>
```

Comment:
You've got almost exactly the same code here to render a form as you do in `new`. You should dry this up with a view partial.